### PR TITLE
[release] 0.1.2: React 16 support

### PR DESCRIPTION
**Pull request by YUzhva**
Hey hi @mrdivyansh 

We are using your outdated package.
Could you please merge that PR and publish the latest changes to the npm?

The only changes here are:
- I added `prop-types` to the _dependencies_ list
- starting to `import PropTypes from 'prop-types'`

Will appreciate fast changes.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -19,7 +19,9 @@
     "clean": "rm -rf build",
     "lint": "eslint src test"
   },
-  "dependencies": {},
+  "dependencies": {
+    "prop-types": "^15.6.2"
```
Comment (mrdivyansh) [inline package.json]: Better to add as peer dependency.

Comment (YUzhva) [reply package.json]: not at all.
`peerDependency` means that people who are using that plugin required to have prop-types.

In our project we are using `flow`, some other could use `TypeScript`.

For example how the "big" plugin managing it:
https://github.com/JedWatson/react-select/blob/master/package.json#L17

Comment (mrdivyansh) [reply package.json]: Make sense.

**Comment by YUzhva** [comment]
Thank you very much!)
Going put back your package into our `package.json` and will be contributing here (=

Have a great week!)

**Pull request merged**